lib/openshift/tuned.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class Tuned < ClusterResource
+    RESOURCE = 'tuned'
```

Comment:
we pluralized the object `RESOURCE`. For this case it would be `RESOUCE='tuneds'`